src/app/fdctl/config_parse.c

```diff
@@ -1,5 +1,7 @@
 #include "config_parse.h"
 
+FD_IMPORT_BINARY( fdctl_default_config, "src/app/fdctl/config/default.toml" );
```

Comment:
The makefile has an explicit link dependency for this you need to update

Reply:
Ah, awesome, we should do this everywhere we use FD_IMPORT